recipe/install_llvm.bat

```diff
@@ -7,5 +9,13 @@ if "%PKG_NAME%" == "llvm-tools" (
     rmdir /S /Q %LIBRARY_LIB%
     rmdir /S /Q %LIBRARY_INC%
     rmdir /S /Q %LIBRARY_PREFIX%\libexec
+) else (
+    REM upstream picks up diaguids.lib from the windows image, see
+    REM https://github.com/llvm/llvm-project/blob/llvmorg-14.0.6/llvm/lib/DebugInfo/PDB/CMakeLists.txt#L17
+    REM which ultimately derives from VSINSTALLDIR, see
+    REM https://github.com/llvm/llvm-project/blob/llvmorg-14.0.6/llvm/cmake/config-ix.cmake#L516
+    REM and gets hardcoded by CMake to point to the path in our windows image.
+    REM This makes it non-portable between image versions (e.g. 2019 vs 2022), so replace
+    REM the hardcoded path with a variable again
+    sed -i "s,C:/Program Files/Microsoft Visual Studio/2022/Enterprise,$ENV{VSINSTALLDIR},g" %LIBRARY_LIB%\cmake\llvm\LLVMExports.cmake
```

Comment:
Can you do a `type %LIBRARY_LIB%\cmake\llvm\LLVMExports.cmake` here so that we can see what the file looks like?

Reply:
I did that a bunch of times during debugging, do you want to keep that in the final version to have it in the log?

Also: before and/or after the sed replacement?

Reply:
If you just want to see what it looks like I can copy the result from one of the previous runs here

Reply:
We can keep it in the final version as well.

Reply:
> Also: before and/or after the sed replacement?

Just after is enough.